README.md

```diff
@@ -2,17 +2,25 @@
 
 # xlsxreader: A Go package for reading data from an xlsx file
 
-## Overview [![GoDoc](https://godoc.org/github.com/thedatashed/xlsxreader?status.svg)](https://godoc.org/github.com/thedatashed/xlsxreader) [![CircleCI](https://circleci.com/gh/TheDataShed/xlsxreader.svg?style=shield&circle-token=278b8226f604014c183d01a96d4eb6ead9d0bc79)](https://github.com/TheDataShed/xlsxreader) [![Go Report Card](https://goreportcard.com/badge/github.com/thedatashed/xlsxreader)](https://goreportcard.com/report/github.com/thedatashed/xlsxreader)
+## Overview
+
+[![Go Reference](https://pkg.go.dev/badge/github.com/thedatashed/xlsxreader.svg)](https://pkg.go.dev/github.com/thedatashed/xlsxreader)
+[![CircleCI](https://circleci.com/gh/TheDataShed/xlsxreader.svg?style=shield&circle-token=278b8226f604014c183d01a96d4eb6ead9d0bc79)](https://github.com/TheDataShed/xlsxreader)
+[![Go Report Card](https://goreportcard.com/badge/github.com/thedatashed/xlsxreader)](https://goreportcard.com/report/github.com/thedatashed/xlsxreader)
 
 A low-memory high performance library for reading data from an xlsx file.
 
-Suitable for reading .xlsx data and designed to aid with the bulk uploading of data where the key requirement is to parse and read raw data.
+Suitable for reading .xlsx data and designed to aid with the bulk uploading of
+data where the key requirement is to parse and read raw data.
 
-The reader will read data out row by row (1->n) and has no concept of headers or data types (this is to be managed by the consumer).
+The reader will read data out row by row (1->n) and has no concept of headers or
+data types (this is to be managed by the consumer).
 
-The reader is currently not concerned with handling some of the more advanced cell data that can be stored in a xlsx file.
+The reader is currently not concerned with handling some of the more advanced
+cell data that can be stored in a xlsx file.
 
-Further reading on how this came to be is available on our [blog](https://www.thedatashed.co.uk/2019/02/13/go-shedsheet-reader/)
+Further reading on how this came to be is available on our
+[blog](https://www.thedatashed.co.uk/2019/02/13/go-shedsheet-reader/)
```

Comment:
What's the logic behind splitting these lines?

Reply:
Wraps to 80 characters so it fits in a terminal window if split it can still be read
easily. To be honest my prettier config just formatted the file so it's fine for us
to revert this.